instance/tests/integration/ansible/requirements.txt

```diff
@@ -1 +1 @@
-git+https://github.com/edx/ansible.git@stable-1.9-plus-edx#egg=ansible==1.9.1-edx.2
```

Comment:
Koa requires 2.8.17 so we might consider bumping this higher.

Reply:
Good idea, I'll do that.